kill.go

```diff
@@ -54,15 +54,15 @@ signal to the init process of the "ubuntu01" container:
 }
 
 func parseSignal(rawSignal string) (unix.Signal, error) {
+	// allow 0 as a valid signal
 	s, err := strconv.Atoi(rawSignal)
 	if err == nil {
 		return unix.Signal(s), nil
 	}
-	sig := strings.ToUpper(rawSignal)
-	if !strings.HasPrefix(sig, "SIG") {
-		sig = "SIG" + sig
```

Comment:
For other reviewers; I was initially thinking we had to keep this (make sure it has a `SIG` prefix), or if this had to be moved to `signal.ParseSignal`, but looks like that's already the case; `signal.ParseSignal()` normalises by _stripping_ the `SIG` prefix (if present);

https://github.com/moby/sys/blob/1a1d2afe36340e8eac900548192ab7a43ac584c8/signal/signal.go#L46

```go
signal, ok := SignalMap[strings.TrimPrefix(strings.ToUpper(rawSignal), "SIG")]
```